Automattic/jetpack
# Blocks: Gathering Tweetstorms

A way to import tweetstorms directly into the block editor.

![tweetstorm](https://user-images.githubusercontent.com/3629020/85486903-ab282e00-b61f-11ea-8a12-05a044ebb275.gif)

#### Changes proposed in this Pull Request:
* Extends the core Twitter embed block, to allow importing tweetstorms.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* New feature. p1HpG7-9mC-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

##### Setup
* Create a [Jurassic Ninja sandbox](https://jurassic.ninja/create/?jetpack-beta&branch=add/tweetstorm-gatherer) against this PR.
* Go through the Jetpack connection process, select a free plan.

##### Testing
* Paste a link to the first tweet in a tweetstorm into a new post.
* Wait for the embed to show.
* Click on the 'Unroll' option in the toolbar.
* Try importing a bunch of different tweetstorms, see what breaks.

#### Proposed changelog entry for your changes:
* Twitter Embeds: Add support for importing tweetstorm content directly into the post.
